config.xml

```diff
@@ -1,29 +1,21 @@
 <?xml version='1.0' encoding='utf-8'?>
 <widget id="io.ionic.starter" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
-    <name>BioPocket</name>
+    <name>MyApp</name>
```

Comment:
Was this overwritten by mistake?

Reply:
Apparently. I didn't changed it manually anyways.